src/ko/toon11/src/eu/kanade/tachiyomi/extension/ko/toon11/Toon11.kt

```diff
@@ -0,0 +1,263 @@
+            document.selectFirst("span:contains(작가) + span")?.also { author = it.text() }
+            document.selectFirst("span:contains(소개) + span")?.also { description = it.text() }
+            document.selectFirst("span:contains(장르) + span")?.also { genre = it.text().split(",").joinToString() }
+        }
+    }
+
+    private fun parseStatus(element: String): Int = when {
+        "완결" in element -> SManga.COMPLETED
+        "주간" in element || "월간" in element || "연재" in element || "격주" in element -> SManga.ONGOING
+        else -> SManga.UNKNOWN
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+        val nav = document.selectFirst("span.pg")
+        val chapters = ArrayList<SChapter>()
+
+        document.select(chapterListSelector()).forEach {
+            chapters.add(chapterFromElement(it))
+        }
+
+        if (nav == null) {
+            return chapters
+        }
+
+        val pg2url = nav.selectFirst(".pg_current ~ .pg_page")!!.absUrl("href")
+
+        // recursively build the chapter list
+
+        fun parseChapters(nextURL: String) {
+            val newpage = fetchPagesFromNav(nextURL)
+            newpage.select(chapterListSelector()).forEach {
+                chapters.add(chapterFromElement(it))
+            }
+            val newURL = newpage.selectFirst(".pg_current ~ .pg_page")?.absUrl("href")
+            if (!newURL.isNullOrBlank()) parseChapters(newURL)
+        }
+
+        parseChapters(pg2url)
+
+        return chapters
+    }
+
+    private fun fetchPagesFromNav(url: String) = client.newCall(GET(url, headers)).execute().asJsoup()
+
+    override fun chapterListSelector() = "#comic-episode-list > li"
+
+    override fun chapterFromElement(element: Element): SChapter {
+        val urlEl = element.selectFirst("button")
+        val dateEl = element.selectFirst(".free-date")
+
+        return SChapter.create().apply {
+            urlEl!!.also {
+                setUrlWithoutDomain(it.attr("onclick").substringAfter("location.href='.").substringBefore("'"))
+                name = it.selectFirst(".episode-title")!!.text()
+            }
+            dateEl?.also { date_upload = dateParse(it.text()) }
+        }
+    }
+
+    private fun dateParse(dateAsString: String): Long {
+        val date: Date? = SimpleDateFormat("yy.MM.dd", Locale.ENGLISH).parse(dateAsString)
+
+        return date?.time ?: 0L
+    }
+
+    override fun pageListRequest(chapter: SChapter): Request {
+        return GET(baseUrl + "/bbs" + chapter.url, headers)
+    }
+
+    override fun pageListParse(document: Document): List<Page> {
+        val pages = mutableListOf<Page>()
+
+        val rawImageLinks = document.selectFirst("script + script[type^=text/javascript]:not([src])")!!.data()
+        val imgList = extractList(rawImageLinks)
+
+        imgList.forEachIndexed { i, img ->
+            pages.add(Page(i, "", "https:$img"))
+        }
+        return pages
```

Comment:
```suggestion
        return imgList.mapIndexed { i, img ->
            pages.add(Page(i, imageUrl = "https:$img"))
        }
```
also remove the mutable list above